packages/wonder-blocks-button/index.js

```diff
@@ -1,4 +1,6 @@
 // @flow
 import Button from "./components/button.js";
+import type {SharedProps} from "./components/button.js";
 
+export type {SharedProps as ButtonProps};
```

Comment:
A number of places are trying to get our Button SharedProps, which don't exist. I think providing this is a good stopgap.

Reply:
Interesting... I hadn't considered people using Props types to construct other Props types for other components.  In a future release we should try to be more consistent about exporting those and thinking about how we're naming them.